beancount_reds_importers/importers/vanguard/__init__.py

```diff
@@ -7,11 +7,18 @@
 
 class Importer(investments.Importer, ofxreader.Importer):
     IMPORTER_NAME = 'Vanguard'
+    # Any memo in the source OFX that's in this set is not carried forward.
+    # Vanguard sets memos that aren't very useful and would create noise in the
+    # generated ledger transactions.
+    SKIP_MEMOS = {
+        "Price as of date based on closing price",
+    }
 
     def custom_init(self):
         self.max_rounding_error = 0.11
         self.filename_pattern_def = '.*OfxDownload'
         self.get_ticker_info = self.get_ticker_info_from_id
+        self.get_payee = self._get_payee
```

Comment:
That makes sense and I also think of this as, out of the box, do people want to see messages like "Price as of date based on closing price"? Personally I can't think of any time to include them so it'd be good to have them suppressed by default. but if you don't agree, I can comment it out.

Reply:
Agree with you: default behavior of suppressing that message makes sense. I think I was more concerned about the capitalization (`.title()`) and other customizations. A vanilla skip-if-found-in-SKIP_MEMOS makes sense to me.

Reply:
I see now. With this I'm also adding the OFX transaction type to the payee field because Vanguard doesn't describe the transaction elsewhere. I then used `title` because it looks nicer. I figured most people will want the additional type information by default too, because the transactions can be inscrutable otherwise.